python_basics/MODULE.bazel

```diff
@@ -13,7 +13,7 @@
 
 module(
     name = "score_python_basics",
-    version = "0.3.1",
+    version = "0.3.2",
```

Comment:
This updates `packaging` to a new major version. That's not a patch release. (After 0.x)